nbextensions/bulkImportCell/paramsWidget.js

```diff
@@ -0,0 +1,551 @@
+define([
```

Comment:
most of this was pulled directly from appParamsWidget, with edits to remove batch mode, and the file inputs. 

the file inputs will have a quite similar flow to how we instantiate the parameter fields, so there may be some more opportunity to further abstract methods as part of that work or once we finish it up.